kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -167,7 +167,12 @@ return {
         current_metric = nil
       end
 
-      return current_metric and current_metric or 0
+      local ok, err = red:set_keepalive(10000, 100)
```

Comment:
We hardcode this elsewhere ^ Perhaps we could develop a separate PR to clean up these keepalive calls across the codebase?